deployments/examples/ocis_oc10_backend/oc10/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM owncloud/server:latest
```

Comment:
Codacy found an issue: [Using latest is prone to errors if the image will ever update. Pin the version explicitly to a release tag](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=6324067)